### DISCUSSION TITLE
blog/2022/220421-use-gzip-bomb-against-scanner/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#2] [Announcements]
# blog/2022/220421-use-gzip-bomb-against-scanner/

Arisa 的个人博客

https://blog.arisa.moe/blog/2022/220421-use-gzip-bomb-against-scanner/

**Comment by InariAimu** [comment]
这个可怜而愚蠢的扫描器

**Comment by MiAcStudio** [comment]
可以用来对抗微信的扫描🤣
